Userland/Libraries/LibJS/Runtime/Object.h

```diff
@@ -99,7 +99,7 @@ class Object : public Cell {
 
     void define_properties(Value properties);
 
-    virtual bool delete_property(const PropertyName&);
+    virtual bool delete_property(PropertyName const&, bool force_throw_exception = false);
```

Comment:
I used a bool here instead of an enum since we might want to call this with `vm.is_in_strict_mode()`